<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matlab_control_showClass</class>
 <widget class="QMainWindow" name="Matlab_control_showClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matlab_control_show_show</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>工件信息：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>平面</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>喇叭口</string>
     </property>
    </item>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_done">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Done</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QtCharts::QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>清除曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartAndStop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>750</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>启动定时器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>select</sender>
   <signal>clicked()</signal>
   <receiver>Matlab_control_showClass</receiver>
   <slot>on_pushButtonSelect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_done</sender>
   <signal>clicked()</signal>
   <receiver>Matlab_control_showClass</receiver>
   <slot>on_pushButtonSelDone_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButtonSelect_clicked()</slot>
  <slot>on_pushButtonSelDone_clicked()</slot>
 </slots>
</ui>
